pom.xml

```diff
@@ -103,6 +102,8 @@
 		<module>artemis-test</module>
 		<module>artemis-cli</module>
 		<module>artemis-gwt</module>
+		<module>artemis-serializer-backends/artemis-json</module>
+		<module>artemis-serializer-backends/artemis-json-libgdx</module>
   </modules>
```

Comment:
I mirrored the gdx setup for backends. Couldn't discern a direct need for adding an extra module , if there is one it would make sense!